src/python/pants/backend/helm/dependency_inference/subsystem.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2024 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+from __future__ import annotations
+
+from enum import Enum
+from functools import cached_property
+
+from pants.option.option_types import EnumOption, StrListOption
+from pants.option.subsystem import Subsystem
+from pants.util.strutil import softwrap
+
+
+class UnownedDependencyUsage(Enum):
```

Comment:
I find this name too wide and generic from a Plugin API point of view, maybe `UnownedHelmDependencyUsage`. I know it doesn't change much from the behavior aspect of the implementation, but it may be less confusing for other plugin authors when they find this type around...

Reply:
True, I copied this from the Python backend. I'm thinking that it might be worth centralising so all backends can use it.